package-lock.json

```diff
@@ -2238,9 +2238,9 @@
           },
           "dependencies": {
             "minimist": {
-              "version": "1.2.5",
-              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
-              "integrity": "sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==",
+              "version": "0.0.8",
```

Comment:
Changing the minimist version to 0.0.8 seems ok.

mkdirp version 0.5.1 needs minimist 0.0.8, not 1.2.5: https://github.com/isaacs/node-mkdirp/blob/d4eff0f06093aed4f387e88e9fc301cb76beedc7/package.json

Reply:
What I don't understand though, is that if I checkout this PR, delete `node_modules/` and run `npm i`, it changes the version back to `1.2.5`.